
Lmod is automatically replacing "cray-mpich/8.1.28" with "openmpi/5.0.0".

Warning: no DISPLAY environment variable.
--No graphics will be displayed.
Traceback (most recent call last):
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/site-packages/netpyne/batch/utils.py", line 538, in evaluator
    simData = json.load(file)['simData']
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 115143 column 16 (char 4587520)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/global/u2/a/adammwea/2DNetworkSimulations/NERSC/batchRun.py", line 231, in <module>
    batchRun(batch_config = batch_config)
  File "/global/u2/a/adammwea/2DNetworkSimulations/NERSC/batchRun.py", line 168, in batchRun
    batch.run()
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/site-packages/netpyne/batch/batch.py", line 283, in run
    evolOptim(self, pc)
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/site-packages/netpyne/batch/evol.py", line 211, in evolOptim
    final_pop = ea.evolve(
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/site-packages/inspyred/ec/ec.py", line 439, in evolve
    initial_fit = evaluator(candidates=initial_cs, args=self._kwargs)
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/site-packages/netpyne/batch/evol.py", line 209, in func
    return evaluator(batch, candidates, args, ngen, pc, **kwargs)
  File "/global/homes/a/adammwea/.conda/envs/2DSims/lib/python3.9/site-packages/netpyne/batch/utils.py", line 552, in evaluator
    print(("%s \n %s" % (err, e)))
OSError: [Errno 122] Disk quota exceeded
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
OSError: [Errno 122] Disk quota exceeded
